assets/base.css

```diff
@@ -1097,14 +1097,15 @@ summary::-webkit-details-marker {
 
   .grid--peek .grid__item {
     width: calc(50% - var(--grid-mobile-horizontal-spacing) - 3rem);
+    min-width: 35%;
   }
 
-  .grid--peek .grid__item:first-of-type {
-    padding-left: 1.5rem;
+  .grid--peek.slider .grid__item:first-of-type {
+    margin-left: 1.5rem;
```

Comment:
The added `.slider` class here is needed for specificity, but should probably be here anyway.

Additionally, I needed to change these to use margin instead of padding so that `slide.offsetLeft` would return the actual left edge of the slide.

Reply:
I'm glad you commented this cause I would have asked. 😄

Reply:
haha I know. Related to this.. I wouldn't say this case warrants a comment or anything (even if they were more commonplace in our practices), but I do worry that retro specificity reduction for specificity reduction's sake (as we tend to do) could/does lead to small things like this being broken in the future.

Reply:
Yeah, I struggle with this sometimes. CSS is really not an easy thing to just read and understand like other code - we have to assume everything is there for a reason, and that the reason may not be entirely obvious, but that also makes it really hard to refactor CSS as well.  It's a fundamental challenge with CSS, and I don't have a good solution to it.